app/models/travel_advice.rb

```diff
@@ -0,0 +1,35 @@
+class TravelAdvice < MultipageContent
+
+  DETAILS_ATTRIBUTES = %i(
+    summary change_description alert_status email_signup_link
+  )
```

Comment:
Rather than metaprogram some attr methods, would it be clearer to define those attrs here on the subclass?